server/exercise/src/main/protocolBuffers/deviceSensorData.proto

```diff
@@ -0,0 +1,28 @@
+//Scala code generated using https://github.com/SandroGrzicic/ScalaBuff/wiki/Generated-code: sbt run --proto_path=./ --scala_out=./
+//Swift code generated using https://github.com/alexeyxo/protobuf-swift: protoc sensorData.proto --swift_out=./
+
+message MultipleDeviceSensorData {
+  enum Source {
+    HEAD = 0;
+    NECK = 1;
+    SHOULDER = 2;
+    CHEST = 3;
+    ARM = 4;
+    FOREARM = 5;
+    HAND = 6;
+    WRIST = 7;
+    WAIST = 8;
+    HIP = 9;
+    THIGH = 10;
+    CALF = 11;
+    ANKLE = 12;
+    FOOT = 13;
+  }
```

Comment:
For now we decided not to. But I assume there are exercises where the movement is not symmetric and thus we may need to do that. I can do that, should be a simple change.